man/systemd-repart.xml

```diff
@@ -251,13 +252,21 @@
         <term><option>--root=</option></term>
 
         <listitem><para>Takes a path to a directory to use as root file system when searching for
-        <filename>repart.d/*.conf</filename> files and for the machine ID file to use as seed. By default
-        when invoked on the regular system this defaults to the host's root file system
+        <filename>repart.d/*.conf</filename> files, for the machine ID file to use as seed and for the
+        <varname>CopyFiles=</varname> and <varname>CopyBlocks=</varname> source files an directories. By
```

Comment:
doesn't matter too much, but is this in the wrong commit? `CopyBlocks` documentation but `--image` commit

Reply:
No, this just extends the `--root=` documentation a bit. We refer to it a bit later in the `--image=` description so I figured it's good to extend it a bit. The stuff I added here only describes what the pre-existing CopyBlocks= setting has to do with `--root=`/`--image=` , it's not about the CopyBlocks=auto special setting then later adds.